src/main/cpp/blaze_util_windows.cc

```diff
@@ -402,7 +402,9 @@ string GetOutputRoot() {
   string home = GetHomeDir();
   if (home.empty()) {
     BAZEL_DIE(blaze_exit_code::LOCAL_ENVIRONMENTAL_ERROR)
-        << "Cannot find a good output root. Use the --output_user_root flag.";
+        << "Cannot find a good output root.\n"
+           "Set the HOME environment variable. Example (in cmd.exe):\n"
+           "    set HOME=c:\\_bazel";
```

Comment:
Is this a good default example? I suppose it will always correctly but I am might not work if there are several users.

You shouldn't use the double slash though. That's actually an invalid path on windows. It should be `set HOME=C:\_bazel`.

Do you think it's worth it to mention that `set` only sets for the current shell? 

Environment variable setting in windows is kinda confusing because `set` will only do the current console, but `setx` (the persistent version) will only do future consoles and has a slightly different syntax (`setx HOME c:\_bazel`, no `=`). I think it's reasonable enough to assume one can look up the persistent command themselves if they are using such a custom setup.

Reply:
> Is this a good default example? I suppose it will always correctly but it might not work if there are several users.

Good point. It seems the `output_root` is only used for computing the `output_user_root`, which does contain the username, so I suppose this is fine, but not trivially so.

> You shouldn't use the double slash though. That's actually an invalid path on windows. It should be `set HOME=C:\_bazel`.

The first `\` just escapes the second one in the string literal.
 
> Do you think it's worth it to mention that `set` only sets for the current shell?

I also thought of that, but no, I don't think so. See below.

> Environment variable setting in windows is kinda confusing because `set` will only do the current console, but `setx` (the persistent version) will only do future consoles and has a slightly different syntax (`setx HOME c:\_bazel`, no `=`). I think it's reasonable enough to assume one can look up the persistent command themselves if they are using such a custom setup.

Exactly. Mentioning that `set` only affects the current session begets the question about setting it for subsequent sessions, and what are the differences in quoting rules of `set` vs `setx`.. and all that is beyond the scope of the error message. The user can google this information if they want.

Reply:
Ah yes of course, the double slash is just because of the string literal. That didn't even occur to me as I've been fiddling around with Dockerfiles and that works differently there. SGTM then!